dsl/pace/dsl/stencil.py

```diff
@@ -298,21 +296,10 @@ def __init__(
 
         stencil_function = gtscript.stencil
         stencil_kwargs = {**self.stencil_config.stencil_kwargs(skip_passes=skip_passes)}
-
-        # Enable distributed compilation if running in parallel
-        if MPI is not None and MPI.COMM_WORLD.Get_size() > 1:
-            stencil_function = future_stencil.future_stencil
-            stencil_kwargs["wrapper"] = self
-        else:
-            # future stencil provides this information and
-            # we want to be consistent with the naming whether we are
-            # running in parallel or not (so we use the same cache)
-            stencil_kwargs["name"] = func.__module__ + "." + func.__name__
+        # stencil_kwargs["name"] = func.__module__ + "." + func.__name__
```

Comment:
I think you want to uncomment this

Reply:
I checked and I think the default is different, so yeah I'll change it back. Thanks!